pkg/share/manager/nextcloud/nextcloud.go

```diff
@@ -403,13 +404,29 @@ func (sm *mgr) GetReceivedShare(ctx context.Context, ref *collaboration.ShareRef
 }
 
 // UpdateReceivedShare updates the received share with share state.
-func (sm *mgr) UpdateReceivedShare(ctx context.Context, ref *collaboration.ShareReference, f *collaboration.UpdateReceivedShareRequest_UpdateField) (*collaboration.ReceivedShare, error) {
+func (sm *mgr) UpdateReceivedShare(ctx context.Context, receivedShare *collaboration.ReceivedShare, fieldMask *field_mask.FieldMask) (*collaboration.ReceivedShare, error) {
+	type updateField struct {
```

Comment:
Thanks for the effort, but we don't want to mimic the old API. In all other places, the Nextcloud-ScienceMesh app API exactly follows the CS3 API, just with a different encoding (JSON instead of protobuf). So when the CS3 API changes, the Nextcloud-ScienceMesh app API just goes along with exactly the same change.